### PR TITLE
lighthouse: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/lighthouse.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# lighthouse
+
+> Analyzes web applications and web pages, collecting modern performance metrics and insights on developer best practices.
+> More information: <https://github.com/GoogleChrome/lighthouse>.
+
+- Generate an HTML report for a specific website and save it to a file:
+
+`lighthouse {{https://example.com}}`
```
Comment (bl-ue) [inline pages/common/lighthouse.md:8]: Can you configure the file it saves to?

File: pages/common/lighthouse.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Generate a JSON report and print to stdout:
```
Comment (bl-ue) [inline pages/common/lighthouse.md:10]:
```suggestion
- Generate a JSON report and print it:
```

File: pages/common/lighthouse.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Generate a JSON report and save it to a specific file:
+
+`lighthouse --output {{json}} --output-path {{path/to/file.json}} {{https://example.com}}`
+
+- Generate a report using the browser in headless mode without logging to stdout:
+
+`lighthouse --quiet --chrome-flags="{{--headless}}" {{https://example.com}}`
+
+- Generate a report using specific JSON HTTP Header key/value pairs to send in requests:
```
Comment (bl-ue) [inline pages/common/lighthouse.md:22]:
```suggestion
- Generate a report, using the HTTP header key/value pairs in the specified JSON file for all requests:
```

File: pages/common/lighthouse.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`lighthouse --extra-headers={{path/to/file.json}} {{https://example.com}}`
+
+- Generate a report only for specific categories:
```
Comment (bl-ue) [inline pages/common/lighthouse.md:26]: `for only` sounds a bit more natual here.
```suggestion
- Generate a report for only specific categories:
```

File: pages/common/lighthouse.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/lighthouse.md:8]: That's the 3rd example.

File: pages/common/lighthouse.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Generate a report using the browser in headless mode without logging to stdout:
+
+`lighthouse --quiet --chrome-flags="{{--headless}}" {{https://example.com}}`
+
+- Generate a report using specific JSON HTTP Header key/value pairs to send in requests:
+
+`lighthouse --extra-headers={{path/to/file.json}} {{https://example.com}}`
+
+- Generate a report for only specific categories:
```
Comment (marchersimon) [inline pages/common/lighthouse.md:26]:
```suggestion
- Generate a report for specific categories only:
```
I thank that sounds a bit better.

**Review by nicokosi — APPROVED** [review]
Seems perfect to me 👍 (but I don't know the command 😇).

File: pages/common/lighthouse.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/lighthouse.md:8]: Then it might be good to say the name of the file that is generated, or at least pattern. What it is?

Comment (navarroaxel) [reply pages/common/lighthouse.md:8]: This is the final command output with the filename:
```
status Generating results... +1ms
Printer html output written to /tmp/www.google.com_2021-06-10_11-39-37.report.html +47ms
ChromeLauncher Killing Chrome instance 11817 +1ms
```

Comment (bl-ue) [reply pages/common/lighthouse.md:8]: How about this then?
```suggestion
- Generate an HTML report for a specific website and save it to a temporary file:

`lighthouse {{https://example.com}}`
```

Comment (bl-ue) [reply pages/common/lighthouse.md:8]: Ah, okay, yes, if that's more accurate it's a lot better 🙂

**Review by marchersimon — APPROVED** [review]
Thanks @navarroaxel

**Review by CleanMachine1 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
I'll assume since as blue sent some suggestions in, this is done. Merging.

**Comment by bl-ue** [comment]
Oh, yes, I'm sorry, I didn't notice that I was requested for review (@CleanMachine1 maybe it would be better to ping me again next time?)

**Review by bl-ue — APPROVED** [review]

**Pull request merged**